modules/imap/functions.php

```diff
@@ -1605,34 +1608,9 @@ function connect_to_imap_server($address, $name, $port, $user, $pass, $tls, $ima
     }
 }
 
-/**
- * @param array $sources
- * @param object $cache
- * @param array $search
- */
-function getCombinedMessagesLists($sources, $cache, $search) {
```

Comment:
This too has been altered. Let me restore it. @amaninyumu1 be careful while doing rebase.